Marlin/Conditionals_post.h

```diff
@@ -794,6 +794,11 @@
     #define MAX_PROBE_X ( SCARA_PRINTABLE_RADIUS)
     #define MIN_PROBE_Y (-SCARA_PRINTABLE_RADIUS)
     #define MAX_PROBE_Y ( SCARA_PRINTABLE_RADIUS)
+  #elif ENABLED(GALVO_XY)
+    #define MIN_PROBE_X (-(GALVOXY_X_WIDTH/2))
```

Comment:
Grouping should be: `(-(GALVOXY_X_WIDTH) / 2)` or just `-(GALVOXY_X_WIDTH) / 2`. Parentheses are needed around any macro that's being multiplied, divided, negated, compared, etc., and in the code we try to be religious about wrapping macros in parentheses, so they're optional around the whole defined value.